torchsummary/torchsummary.py

```diff
@@ -63,6 +60,8 @@ def hook(module, input, output):
         line_new = '{:>20}  {:>25} {:>15}'.format('Layer (type)', 'Output Shape', 'Param #')
```

Comment:
You can make a general line, like `line_thresholds='{:>20}  {:>25} {:>15}'` and then simply format in next calls